lib/go-atscfg/remapdotconfig_test.go

```diff
@@ -1599,16 +1599,16 @@ func TestMakeRemapDotConfigMidSlicePluginRangeRequestHandling(t *testing.T) {
 	ds.Active = util.BoolPtr(true)
 
 	ds2 := DeliveryService{}
-	ds2.ID = util.IntPtr(48)
+	ds2.ID = util.IntPtr(50)
 	dsType2 := tc.DSType("HTTP_LIVE_NATNL")
 	ds2.Type = &dsType2
-	ds2.OrgServerFQDN = util.StrPtr("origin.example.test")
+	//ds2.OrgServerFQDN = util.StrPtr("origin2.example.test")
```

Comment:
Why'd you comment this out?

Reply:
probably testing and committed by accident. fixed.